packages/lib/models/Setting.ts

```diff
@@ -915,7 +915,7 @@ class Setting extends BaseModel {
 			// Deprecated in favour of windowContentZoomFactor
 			'style.zoom': { value: 100, type: SettingItemType.Int, public: false, storage: SettingStorage.File, appTypes: [AppType.Desktop], section: 'appearance', label: () => '', minimum: 50, maximum: 500, step: 10 },
 
-			'style.editor.fontSize': { value: 13, type: SettingItemType.Int, public: true, storage: SettingStorage.File, appTypes: [AppType.Desktop], section: 'appearance', label: () => _('Editor font size'), minimum: 4, maximum: 50, step: 1 },
+			'style.editor.fontSize': { value: 15, type: SettingItemType.Int, public: true, storage: SettingStorage.File, appTypes: [AppType.Desktop], section: 'appearance', label: () => _('Editor font size'), minimum: 4, maximum: 50, step: 1 },
```

Comment:
The problem with this change is that if people never changed this setting, because they were happy with the default, the text will be much bigger after this PR.

I think there should be an extra entry in the release notes when a default value changes.

Reply:
Yes that makes sense, we can add an entry about it in the release notes.

Reply:
Here's some text you can use:

Attention: The default font size has been changed from 13 to 15px in the Markdown editor (CodeMirror). You can change the font size in `Config > Appearance > Editor font size`.